packages/integration-tests/test/challengeExitTest.js

```diff
@@ -45,88 +46,157 @@ describe('Challenge exit tests', async () => {
+    const challengeData = await childChain.getChallengeData(invalidExit.details.utxo_pos)
+    assert.hasAllKeys(challengeData, ['input_index', 'output_id', 'sig', 'txbytes'])
+    receipt = await rootChain.challengeStandardExit(
+      challengeData.output_id,
       challengeData.txbytes,
-      challengeData.proof,
-      challengeData.sigs,
-      bobAccount.privateKey
+      challengeData.input_index,
+      challengeData.sig,
+      {
+        privateKey: bobAccount.privateKey,
+        from: bobAccount.address
+      }
     )
     console.log(`Bob called RootChain.challengeExit(): txhash = ${receipt.transactionHash}`)
 
+    // Keep track of how much Bob spends on gas
+    let bobSpentOnGas = await helper.spentOnGas(web3, receipt)
+
     // Alice waits for the challenge period to be over...
     console.log(`Waiting for challenge period... ${CHALLENGE_PERIOD}ms`)
     await helper.sleep(CHALLENGE_PERIOD)
 
     // ...and calls finalize exits.
-    receipt = await rootChain.finalizeExits(aliceAccount.address, ETH_CURRENCY, 0, 1, aliceAccount.privateKey)
-    console.log(`Alice called RootChain.finalizeExits(): txhash = ${receipt.transactionHash}`)
+    receipt = await rootChain.processExits(
+      ETH_CURRENCY,
+      0,
+      1,
+      {
+        privateKey: aliceAccount.privateKey,
+        from: aliceAccount.address
+      }
+    )
+    console.log(`Alice called RootChain.processExits(): txhash = ${receipt.transactionHash}`)
+    aliceSpentOnGas.iadd(await helper.spentOnGas(web3, receipt))
 
+    // Get Alice's ETH balance
     // Get Alice's ETH balance
     const aliceEthBalance = await web3.eth.getBalance(aliceAccount.address)
-    // Alice's dishonest exit did not successfully complete, so her balance should
-    // still be lower than INTIIAL_ALICE_AMOUNT - DEPOSIT_AMOUNT (it's lower because of gas spent)
-    const expected = web3.utils.toBN(INTIIAL_ALICE_AMOUNT).sub(web3.utils.toBN(DEPOSIT_AMOUNT))
-    assert.isBelow(Number(aliceEthBalance), Number(expected.toString()))
+    // Alice's dishonest exit did not successfully complete, so her balance should be
+    // INTIIAL_ALICE_AMOUNT - DEPOSIT_AMOUNT - STANDARD_EXIT_BOND - gas spent
+    const expected = web3.utils.toBN(INTIIAL_ALICE_AMOUNT)
+      .sub(web3.utils.toBN(DEPOSIT_AMOUNT))
+      .sub(web3.utils.toBN(rootChain.getStandardExitBond()))
+      .sub(aliceSpentOnGas)
+    assert.equal(aliceEthBalance.toString(), expected.toString())
+
+    // Bob successfully challenged the exit, so he should have received the exit bond
+    const bobEthBalance = await web3.eth.getBalance(bobAccount.address)
+    const bobExpectedBalance = web3.utils.toBN(INTIIAL_BOB_AMOUNT)
+      .add(web3.utils.toBN(rootChain.getStandardExitBond()))
+      .sub(bobSpentOnGas)
+    assert.equal(bobEthBalance.toString(), bobExpectedBalance.toString())
   })
 
-  it('should exit dishonestly if not challenged', async () => {
+  // Skip this test as it leaves the childchain byzantine
```

Comment:
What was happening with this test before? Does it pass and then following tests fail?